app/src/main/java/ch/sdp/vibester/user/User.kt

```diff
@@ -9,5 +9,6 @@ data class User(
     var bestScore: Int = 0,
     var correctSongs: Int = 0,
     var ranking: Int = 0,
-    var uid: String = ""
+    var uid: String = "",
+    var friends: Map<String,Boolean> = mapOf()
```

Comment:
It will be nice to have some documentation about this class like what's the string and boolean represent.

Reply:
done!